scripts/tests/test_zkai_proof_pressure_wide_grid_selector_gate.py

```
@@ -0,0 +1,96 @@
+            json_path = tmp_path / "wide-grid.json"
+            tsv_path = tmp_path / "wide-grid.tsv"
+            gate.write_json(json_path, self.payload)
+            gate.write_tsv(tsv_path, self.payload)
+            self.assertEqual(json.loads(json_path.read_text(encoding="utf-8"))["schema"], gate.SCHEMA)
+            tsv = tsv_path.read_text(encoding="utf-8")
+            self.assertIn("d64_h2_seq32", tsv)
+            self.assertIn("FIRST_FALSIFICATION_ROW", tsv)
+
+    def test_write_outputs_reject_absolute_outside_evidence_dir(self):
+        with tempfile.TemporaryDirectory() as tmp:
+            with self.assertRaisesRegex(gate.ProofPressureWideGridSelectorError, "stay inside evidence dir"):
```

Ruff (0.15.13)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)